Android/Fundamentals/drawables-ii/create-a-shape.md

```diff
@@ -0,0 +1,121 @@
+# create-a-shape
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+A `shape drawable` is an XML that defines geometric shapes including colors, borders, and gradients which can get assigned to Views.  Drawable shapes are used to create complex shapes that can attached as a background to a layout or some view on the screen. A shape drawable is useful to change the shape, border, and gradient of a button background as an example. 
+
+The shape can be described as properties such as corners for rounding, gradient for backgrounds, padding for spacing, solid for background colors, and stroke for border.
+
+**To define shape:**
+1.	Create a new Android XML file in the folder res/drawable.
+2.	Make sure the root element of the file is <shape> as this defines a ShapeDrawable.
+3.  Once the shape is defined, you can specify it as a background resource to any view: android:background=”@drawable/myshape”.
+
+**Example**
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android=
+"http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+  
+    <stroke
```

Comment:
then add this shape definition, maybe take a screenshot of this shape and note what it would produce on the screen